### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by lipalath-ms** [pnp/powershell#1787] [General]
```
Connect-PnPOnline -Url $sharePointUSSiteUrl -Credentials $credential -TenantAdminUrl $sharePointUSTenantUrl
Add-PnPApp -Path $Path -Scope Tenant -Overwrite -Publish
```
is giving the error: 

> Connect-PnPOnline : The sign-in name or password does not match one in the Microsoft account system.

I followed this article: https://www.sharepointdiary.com/2019/03/connect-pnponline-with-appid-and-appsecret.html
and with that I see the error: 

>The remote server returned an error: (401) Unauthorized. 

On following this article https://docs.microsoft.com/en-us/sharepoint/dev/solution-guidance/security-apponly-azuread, I see this error:

>Connect-PnPOnline : Exception has been thrown by the target of an invocation.

What am I missing?

**Comment by kpinel** [comment]
Experiencing the same issue and have confirmed that the password is correct.
The login window freezes even when using ForceAuthentication. 
Have temporarily disabled MFA and waiting for that to take effect.
-UseWebLogin works but there are some cmdlets that don't seem to work properly.

This was working a few weeks ago as we provision several sites per week from automation.
